### ISSUE TITLE
[Desktop] [Mobile] Feed screen | Filter section spelling mistakes

### ISSUE THREAD

**Issue by galeza** [FightPandemics/FightPandemics#487]
**Question was sent to content writer J Cameron.**

env: https://staging.fightpandemics.work/feed

1. Go to Feed screen
2. Open filters section

a. Should we have From who or from whom?
Figma has both options:
![Screen Shot 2020-05-26 at 21 25 15](https://user-images.githubusercontent.com/21987083/82941895-703ac800-9f97-11ea-9dc0-9866d2e5bf82.png)
Applications:
![Screen Shot 2020-05-26 at 21 25 49](https://user-images.githubusercontent.com/21987083/82941919-7d57b700-9f97-11ea-99aa-42fcb4470046.png)

b. **Whom** should start with capital or lower letter

c. In the **From whom** section: option should be Others or Other?
From 
![Screen Shot 2020-05-26 at 21 26 34](https://user-images.githubusercontent.com/21987083/82942033-abd59200-9f97-11ea-8ee3-bf5ff4bec3d9.png)

d. In the **Type** section: option should be Others or Other?
Application we have other:
![Screen Shot 2020-05-26 at 21 29 41](https://user-images.githubusercontent.com/21987083/82942322-12f34680-9f98-11ea-9425-a0792880e102.png)
Figma we have others:
![Screen Shot 2020-05-26 at 21 29 51](https://user-images.githubusercontent.com/21987083/82942358-1edf0880-9f98-11ea-972e-d9c23050b11d.png)

e. Button should be View Result or View Results:
Application View Results (with capital letter)
![Screen Shot 2020-05-26 at 21 32 47](https://user-images.githubusercontent.com/21987083/82942585-78473780-9f98-11ea-8d1c-cd5398665814.png)
Figma View result (lower letter, no plural)
![Screen Shot 2020-05-26 at 21 33 22](https://user-images.githubusercontent.com/21987083/82942631-8bf29e00-9f98-11ea-9836-f1690bf7c542.png)

**Comment by omarzoghayyer** [comment]
I can help out with cleaning this up.

**Comment by omarzoghayyer** [comment]
@galeza This issue has been resolved. In addition to that, I fixed 2 more silly mistakes I just ran into (look at the screenshot) 1. I deleted the space between the last letter and first dot. "Something went wrong ..." To "Something went wrong..."
2. "Create post" to "Create a post"
<img width="789" alt="Screen Shot 2020-05-26 at 2 31 57 PM" src="https://user-images.githubusercontent.com/49251966/82952340-a9564680-9f5d-11ea-9591-8b9d0b92acbd.png">


Also, Thank you for always being descriptive. It makes the job easier :)
